packages/graphql-cli-graphback-plugin/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "graphql-cli-graphback-plugin",
+  "version": "0.2.1",
+  "description": "Auto generate resolvers on top of the graphql and database of your choice",
+  "homepage": "https://github.com/ankitjena/graphql-cli-graphback-plugin",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "typescript": {
+    "definition": "types/index.d.ts"
+  },
+  "author": "wtrocki@redhat.com",
+  "scripts": {
+    "clean": "rimraf coverage dist types",
+    "build": "npm run clean && tsc",
+    "cmd": "node ./dist/index.js",
+    "watch": "tsc -w",
+    "lint": "tslint -t stylish --project \"tsconfig.json\""
+  },
+  "engines": {
+    "node": "12.9.1"
```

Comment:
minor nit: this is inaccurate. I can't remember the exact syntax but it should probably be something like `> 8.x.x`. Refer to npm's `package.json` reference documentation to find out the correct syntax